pkg/machines/components/networks/createNetworkDialog.css

```diff
@@ -0,0 +1,11 @@
+.create-network-dialog-grid {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
```

Comment:
Also, you may want to consider adding `ct-form` to `create-network-dialog-grid` or changing the `1fr 1fr` to `auto 1fr` here.

Doing either should make it so there isn't such a big gap between the labels and the forms.